.github/workflows/rust.yml

```
@@ -23,13 +23,7 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Cache dependencies
-      uses: actions/cache@v2
-      with:
-        path: |
-          ~/.cargo
-          ./Cargo.lock
-          ./target
-        key: ${{ matrix.os }}-${{ matrix.rust }}-${{ hashFiles('Cargo.toml') }}
+      uses: Swatinem/rust-cache@c5ed9ba6b7e1bb8aff90d43acd2f0af4990fa57c
```

```suggestion
      uses: Swatinem/rust-cache@v1
```

It is possible to use `@v1` instead? :)

---

I'd not go for it 😄 

> I use the v1 branch similar to master development, so if you want to have a more stable experience, please use a fixed revision or tag.

(https://github.com/Swatinem/rust-cache#versioning)

---

Can we use a version then, I mean a more human readable version, like v1.3.0? :)

---

yep, done!